src/EnergyPlus/test_ep_as_library.cc

```diff
@@ -63,16 +62,17 @@ void progress_callback_handler(int const progress)
 int main(int argc, char *argv[])
 {
     std::cout << "Using EnergyPlus as a library." << std::endl;
-    EnergyPlus::EnergyPlusData state;
-    StoreMessageCallback(state, message_callback_handler);
-    StoreProgressCallback(state, progress_callback_handler);
+    EnergyPlus::EnergyPlusData *state{reinterpret_cast<EnergyPlus::EnergyPlusData *>(stateNew())};
+    StoreMessageCallback(*state, message_callback_handler);
+    StoreProgressCallback(*state, progress_callback_handler);
 
     int status(EXIT_FAILURE);
     if (argc < 2) {
         std::cout << "Call this with a path to run EnergyPlus as the only argument" << std::endl;
         return EXIT_FAILURE;
     } else {
-        status = RunEnergyPlus(state, argv[1]);
+        status = RunEnergyPlus(*state, argv[1]);
+        stateDelete(reinterpret_cast<EnergyPlusState>(state));
```

Comment:
And if you declare the state variable as an `EnergyPlusState` alias type, then this should be doable with just `stateDelete(state)`, no reinterpret necessary.